js-api-doc/compile.d.ts

```diff
@@ -140,10 +140,15 @@ export class AsyncCompiler {
  *
  * This only allows synchronous {@link Importer}s and {@link CustomFunction}s.
  *
- * **Heads up!** When using the [sass-embedded] npm package,
- * **{@link compileAsync} is almost always faster than {@link compile}**, due to
- * the overhead of emulating synchronous messaging with worker threads and
- * concurrent compilations being blocked on main thread.
+ * **Heads up!** When using the [sass-embedded] npm package for single
+ * compilations, **{@link compileAsync} is almost always faster than
+ * {@link compile}**, due to the overhead of emulating synchronous messaging
+ * with worker threads and concurrent compilations being blocked on main thread.
+ *
+ * If you are running multiple compilations with the [sass-embedded] npm
+ * package, using an {@link AsyncCompiler} or a {@link Compiler} will be much
+ * faster on average, and the difference between synchronous and asynchronous
+ * compilation will likely not be significant.
```

Comment:
Is this true? If we use `Compiler` doesn't it means it has to compile files sequentially, while `AsyncCompiler` can initialize multiple Promises concurrently.

Reply:
Ok, I saw the benchmark code and apparently it did not consider the case where you can start multiple promises concurrently on `AsyncCompiler ` but you cannot start multiple compilations Compiler concurrently.

E.g. The following code shows the true benefit of concurrent compilations with async compiler, which sync compiler will not be able to compete.

``` js
const asyncCompiler = await initAsyncCompiler();
await Promise.all([
  asyncCompiler.compileAsync(path),
  asyncCompiler.compileAsync(path),
  asyncCompiler.compileAsync(path),
  asyncCompiler.compileAsync(path)
]);
```